pkgs/run-sotest/src/run_sotest/poll_app.py

```diff
@@ -0,0 +1,33 @@
+#######################################
+#                                     #
+#  Copyright Cyberus Technology GmbH  #
+#           All rights reserved       #
+#                                     #
+#######################################
+
+import argparse
+
+from .poll_test_run import poll_test_run
+
+
+def parse_cmdline_args():
+    """Command line parsing"""
+    parser = argparse.ArgumentParser(description="Sotest Test Creator")
+    parser.add_argument("sotest_url", help="Base URL to SoTest's web UI")
+    parser.add_argument("--id", required=True, help="URL to be linked to in the web UI")
```

Comment:
The help text looks like a copy/paste mistake ;)

Reply:
Fixed, thanks :)